src/profile/notification-settings/utils.ts

```diff
@@ -0,0 +1,62 @@
+import { refreshSamUserAttributes, refreshTerraProfile } from 'src/auth/auth';
+import { Ajax } from 'src/libs/ajax';
+import Events, { EventWorkspaceAttributes, extractWorkspaceDetails } from 'src/libs/events';
+
+export const workspaceSubmissionNotificationKeys = (namespace: string, name: string) => [
+  `notifications/SuccessfulSubmissionNotification/${namespace}/${name}`,
+  `notifications/FailedSubmissionNotification/${namespace}/${name}`,
+  `notifications/AbortedSubmissionNotification/${namespace}/${name}`,
+];
+
+export const workspaceChangedNotificationKey = (namespace: string, name: string) => [
+  `notifications/WorkspaceChangedNotification/${namespace}/${name}`,
+];
+
+export const workspaceSubmissionNotificationInfo =
+  'Receive email notifications when a submission in this workspace has succeeded, failed, or been aborted.';
+export const workspaceChangedNotificationInfo =
+  'Receive email notifications when an owner of a workspace sends a workspace changed notification.';
```

Comment:
I tried to be explicit here about the fact that a workspace owner has to choose to push out a notification-- I wanted to avoid people thinking that they should get notifications for every workspace. I am certainly open to wording suggestions though!

Reply:
`'Receive emails for notifications from the owner on workspace changes'`?

Reply:
I'm trying to convey the idea that the owner has to push them out… and thus that one shouldn't expect to get notifications in general. I'm just worried that people are going to think "I have this enabled, why aren't I receiving notifications"?